packages/@pollyjs/adapter-xhr/tests/integration/adapter-test.js

```diff
@@ -63,6 +64,48 @@ describe('Integration | XHR Adapter', function() {
 
     expect(abortEventCalled).to.equal(true);
   });
+
+  function testDownloadingBinaryData(responseType) {
+    it(`should be able to download binary content (${responseType})`, async function() {
+      this.timeout(10000);
+
+      const fetch = async () =>
+        Buffer.from(
+          await this.fetch('https://placekitten.com/32/32', {
```

Comment:
Should we use our local test server instead of external?  I've had issues in the past and had to skip tests that were hitting external APIs that became flaky over time

Reply:
Updated in the `fetch-adapter` PR, will use the new binary middleware when it gets merged.